MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl

```diff
@@ -301,3 +301,451 @@ resource configservice_label system_audit_trail_view_change_history_tooltip {
+    },
+    {
+      default_translation = 'Trænode'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label system_audit_trail_filter_metafield_group {
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_METAFIELD_GROUP'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Metafield group'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Metafelt-gruppe'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label system_audit_trail_filter_asset_type_extension {
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_ASSET_TYPE_EXTENSION'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Asset type extension'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Assettype-extension'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label system_audit_trail_filter_sso_configuration {
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_SSO_CONFIGURATION'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'SSO configuration'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'SSO-konfiguration'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label system_audit_trail_filter_automation {
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_AUTOMATION'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Automation'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Automation'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label system_audit_trail_filter_business_workflow_definition {
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_BUSINESS_WORKFLOW_DEFINITION'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Workflow definition'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Workflow-definition'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label system_audit_trail_filter_business_workflow_instance {
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_BUSINESS_WORKFLOW_INSTANCE'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Workflow instance'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Workflow-instans'
```

Comment:
Task?

Reply:
I don't know, originally on backend this filter calls 'Business Workflow Instance'